### PR TITLE
added Bradley-Terry models

### PR DESCRIPTION
**Pull request by jvparidon**
Added Bradley-Terry models since they can be restated as a special case of the GLMM with multiple membership.

Also renamed "multimembership models" to "multiple membership models" since that seems to be the more common term across the literature and makes it easier for people to find what they're looking for.

### CODE REVIEW

File: MixedModels.md [hunk 1]
```diff
@@ -100,14 +100,15 @@ General estimating equations (GEEs) are an alternative approach to fitting clust
 - [**Additive models**]{#additive-models} (models incorporating smooth functional components such as regression splines or Gaussian processes): `r pkg("gamm4")`, `r pkg("mgcv")`, `r pkg("brms", priority = "core")`, `r pkg("lmeSplines")`, `r pkg("bamlss")`, `r pkg("gamlss")`, `r github("Biometris/LMMsolver")`, `r pkg("R2BayesX")`, `r pkg("GLMMRR")`.
 - **Big data/distributed computation**: `r pkg("lmmpar")`, `r pkg("mbest")`. See also [MixedModels.jl](https://juliastats.org/MixedModels.jl/dev/) (Julia), [diamond](https://github.com/stitchfix/diamond) (Python).
 - **Bioinformatics/quantitative genetics**: `r pkg("MCMC.qpcr")`, `r pkg("QGglmm")`, `r pkg("CpGassoc")` (methylation studies).  
+- **Bradley-Terry models**: The Bradley-Terry model can be parameterized as a special case of the GLMM with multiple membership and fitted with `r github("jvparidon/lmerMultiMember")`.
```
Comment (palday) [inline MixedModels.md:103]:
```suggestion
- **Bradley-Terry models**: The Bradley-Terry model can be parameterized as a special case of the [GLMM with multiple membership](#multimembership-models). `r github("jvparidon/lmerMultiMember")` provides convenience functions for expressing the Bradley-Terry model.
```

**Review by palday — APPROVED** [review]

**Comment by jpiaskowski** [comment]
Do we really need a separate section for Bradley terry models when there is only one example and it fits within another category?

**Comment by jvparidon** [comment]
Fair question!  
Someone emailed R-sig-mixed-models to ask about Bradley-Terry models in a GLMM framework last week. There's also a StackOverflow question asking about it. I took that as a sign that there's at least some existing interest in these models, so I thought this might help people find their way around.

**Comment by jpiaskowski** [comment]
How about we keep the where it's mentioned already and include a comment that it can fit a Bradley Terry model? That what a person can still find it via key word searching?

**Comment by zeileis** [comment]
I think it would make more sense to suggest this to the "Psychometrics" task view. This covers Bradley-Terry models already and it already points out that various types of Rasch models can be fitted via GLMMs, so adding a similar comment for Bradley-Terry models might be helpful. @pmair78

**Comment by jvparidon** [comment]
I'd be happy to submit a PR implementing the change @jpiaskowski suggested.

Moving the whole thing to the Psychometrics task view makes no sense to me, because A) Bradley-Terry models have applications outside of psychometrics, and B) if I were looking to fit a Bradley-Terry model as a a mixed-effects models, the Mixed Models task view is where I would come looking.

**Comment by zeileis** [comment]
Let's see what @pmair78 says. If he adopts it for the Psychometrics task view (which would make sense in any case in my opinion) I wouldn't recommend duplicating the contents in the MixedModels task view. One could add a brief pointer from MixedModels to Psychometrics, though.

**Comment by jpiaskowski** [comment]
I drafted this up:

- [**Multiple membership models**]{#multimembership-models}: (Bayesian) `r pkg("MCMCglmm", priority = "core")`, `r pkg("brms", priority = "core")`, `r github("benrosche/rmm")`; (frequentist) `r github("jvparidon/lmerMultiMember")` (can also fit the Bradley-Terry model). 

Agreed with @jvparidon tho - for ease of helping people find this. I just don't think it needs its own paragraph.

**Comment by pmair78** [comment]
As @zeileis suggested, I will add a line in the psycho ctv as this is relevant. Don't see any issue with having and entry in MixedModels too, as suggested by @jvparidon

**Pull request merged**